docs/content/en/docs/clustermgmt/observability/cluster-verify.md

```diff
@@ -1,122 +1,44 @@
-kube-system                         coredns-69797695c4-kdtjc                                         1/1     Running   0             3d22h
-kube-system                         coredns-69797695c4-r25vv                                         1/1     Running   0             3d22h
-kube-system                         etcd-mgmt-clrt4                                                  1/1     Running   0             3d22h
-kube-system                         kube-apiserver-mgmt-clrt4                                        1/1     Running   0             3d22h
-kube-system                         kube-controller-manager-mgmt-clrt4                               1/1     Running   0             3d22h
-kube-system                         kube-proxy-588gj                                                 1/1     Running   0             3d22h
-kube-system                         kube-proxy-hrksw                                                 1/1     Running   0             3d22h
-kube-system                         kube-scheduler-mgmt-clrt4                                        1/1     Running   0             3d22h
-kube-system                         kube-vip-mgmt-clrt4                                              1/1     Running   0             3d22h
-kube-system                         vsphere-cloud-controller-manager-7vzjx                           1/1     Running   0             3d22h
-kube-system                         vsphere-cloud-controller-manager-cqfs5                           1/1     Running   0             3d22h
+kubectl get pods -n kube-system | head -n 5
+NAME                                     READY   STATUS    RESTARTS      AGE
+kube-apiserver-eksapoc-lkfnr             1/1     Running   0             10d
+kube-apiserver-eksapoc-wjq2c             1/1     Running   0             10d
+kube-apiserver-eksapoc-zqsv9             1/1     Running   0             10d
+kube-controller-manager-eksapoc-lkfnr    1/1     Running   0             10d
+kube-controller-manager-eksapoc-wjq2c    1/1     Running   0             10d
 ```
 
 To verify that the expected number of cluster nodes are up and running, use the `kubectl` command to show that nodes are `Ready`.
 
-This will confirm that the expected number of nodes are present.
-Worker nodes are named using the cluster name followed by the worker node group name (example: mgmt-md-0), the others are control plane nodes.
-
-```
-kubectl get nodes
-NAME                              STATUS   ROLES           AGE   VERSION
-mgmt-clrt4                        Ready    control-plane   3d22h   v1.27.1-eks-61789d8
-mgmt-md-0-5557f7c7bxsjkdg-l2kpt   Ready    <none>          3d22h   v1.27.1-eks-61789d8
-```
-
-**Verify an EKS Anywhere cluster from the management cluster**
-
-Set up `CLUSTER_NAME` and `KUBECONFIG` environment variable for the management cluster:
 ```
-export CLUSTER_NAME=mgmt
-export KUBECONFIG=${CLUSTER_NAME}/${CLUSTER_NAME}-eks-a-cluster.kubeconfig
+kubectl get nodes | grep -i control-plane
```

Comment:
We need to have the command and output in separate code blocks for easier copy / paste